formbricks/formbricks
# feat: add GTM integration in survey page

## What does this PR do?



Feat: include query parameter from current page to redirected page for marketing needs like tracking url

Feat: add GTM integration on survey page



## Type of change




## How should this be tested?



- Create Survey and access the survey url with utm you can build utm url here https://utmbuilder.net/
- in survey setting you should set the Redirect on completion on your desired website
- when you already submit the survey the utm (query parameter) should be included on the redirected url

## Checklist



### Required


### Appreciated
